smack-extensions/src/main/java/org/jivesoftware/smackx/muc/MultiUserChat.java

```diff
@@ -621,6 +621,7 @@ public MucCreateConfigFormHandle createOrJoinIfNecessary(Resourcepart nickname,
      * joining the room, the room will decide the amount of history to send.
      *
      * @param nickname the nickname to use.
+     * @return the leave presence as reflected by the MUC.
```

Comment:
XEP-0045 uses the term "self-presence", hence I think this should also read self-presence (instead of just "presence).